core/storage/email/gae_models.py

```diff
@@ -103,6 +127,7 @@ def create(
         email_model_instance.put()
 
     def put(self):
+    	"""Put the email instance into the messages by hashing it."""
```

Comment:
I don't understand what this means, and I don't think it accurately reflects what this method is doing.